package.json

```diff
@@ -2,7 +2,7 @@
   "name": "web.dev",
   "version": "3.0.0",
   "scripts": {
-    "algolia": "node algolia.js",
+    "algolia": "node -e \"require('./algolia.js').index()\"",
```

Comment:
Is this because you changed algolia to a module? 
Then create another script, called e.g runAlgolia, and keep index logic there rather than inline in package.json.